docs/reference/modules/util.md

```diff
@@ -27,7 +27,7 @@ Because this module has an alias, you can call these from another module from th
 
 ### Logging utilities
 
-The descriptions for `log`, `info`, `debug`, `warn`, and `error` below reflect default behavior. See the `logger` option description above for information about custom behavior.
+The descriptions for `log`, `info`, `debug`, `warn`, and `error` below reflect default behavior. See the `logger` option description above for information about custom behavior. The `info`, `debug`, `warn`, and `error` methods have been deprecated going forward in favor of the methods exposed by the [`@apostrophecms/log` module](/reference/modules/log.html).
```

Comment:
in favor of the structured logging methods (with a quick list) now available on every module.

Reply:
Changed.